build.gradle

```diff
@@ -192,10 +192,6 @@ jacocoTestReport {
     }
 }
 
-task wrapper(type: Wrapper) {
-    gradleVersion = '4.2'
```

Comment:
why did you remove this section? it was convenient for me to see current version and be able to update Gradle this way

Reply:
Wrapper is built-in to Gradle now. If you run gradle wrapper on a new/existing project, it will set the wrapper version to the version of gradle you have installed.

There's an option you can pass to set the version explicitly. `gradle wrapper --gradle-version 4.6`

So these are redundant lines.

The easiest way to see/update the version is to look in the gradle-wrapper.properties file. And that's all the task is doing anyway...

Reply:
Yes, the `wrapper` task is built-in now. Removing these redundant lines will avoid having to modify `build.gradle` during a Gradle wrapper upgrade.

Three easy ways to check the version of the used Gradle wrapper:

1. Open `gradle-wrapper.properties` in an editor, check `distributionUrl`
1. `cat gradle/wrapper/gradle-wrapper.properties | grep Url`
1. `./gradlew --version`

Reply:
I like my old ways, but okay, i feel convinced.